pegasus/sites.v3/code.org/public/thebadguys.md.partial

```diff
@@ -18,12 +17,15 @@ theme: responsive
 {{ clearfix }}
 
 
-
 ## Keep Learning
 
 Enjoyed creating adventures with the Bad Guys? Keep the creativity going with other engaging activities: Use code to choreograph a dance to today's top hits, add moody visuals to a poem, or even clean up the ocean!
 
-
+{{ featured_tutorial, title: "Food", body: "Use your creativity and problem solving skills to explore and build underwater worlds with code!", image: "images/csc/helloworld/cschelloworld_happyfood2.gif", url: "https://studio.code.org/projects/spritelab/sC_ZiNi_x5GUqsWHE2M4CrcbjU8XvtD3VNT7TM0Y0N8" }}
+{{ featured_tutorial, title: "Emoji", body: "Featuring Katy Perry, Shawn Mendes, Panic! At The Disco, Lil Nas X, Jonas Brothers, Nicki Minaj, and 34 more!", image: "images/csc/helloworld/cschelloworld_emoji.gif", url: "https://studio.code.org/projects/spritelab/9HGWXijqhLzaIIUQbPXlNmWgMO1SXzf3TvMHNtbOXmc" }}
+{{ featured_tutorial, title: "Animals", body: "Body text goes here.", image: "images/csc/helloworld/cschelloworld_animals.gif", url: "https://studio.code.org/projects/spritelab/rYH8D8eAvWOjuiOpWbHzN4HAtis4ykKTIjGcNPP9zD4" }}
+{{ featured_tutorial, title: "Retro", body: "Body text goes here.", image: "images/csc/helloworld/cschelloworld_retro.gif", url: "https://studio.code.org/projects/spritelab/rYH8D8eAvWOjuiOpWbHzN7yo2E1S0q87VqlzaBz7oqo" }}
```

Comment:
Just curious: is it normal for files like this to require so much horizontal scrolling. Why don't these wrap to new lines? I'm seeing it through the file and not just with your changes.

Reply:
I don't think this particular format (which I added last year) currently supports newlines, though it would obviously help readability.